coalib/bears/Bear.py

```diff
@@ -24,6 +24,26 @@
 from .meta import bearclass
 
 
+def _debugger_setting_value(bear):
+    """
+    Determines the value of ``debugger`` setting.
+
+    :param bear: Bear object.
+    :return:     True if ``debug_bears`` is ``True`` or if bear name specified
+                 in ``debug_bears`` setting match with the bear parameter.
+    """
+    if bear is None:
+        return False
+    if 'debug_bears' not in bear.section:
+        return False
+    try:
+        return bool(bear.section['debug_bears'])
+    except ValueError:
+        pass
+    return bear.name.lower() in (
+            map(str.lower, list(bear.section['debug_bears'])))
```

Comment:
Hm maybe catch here a `ValueError` as well, and emit a better descriptive message, since bools and string-lists are allowed in that setting. What do you think?
-->
```python
try:
    return bear.name.lower() in ...
except ValueError:
    raise ValueError('could not parse setting "debug_bears" into bool or string-list')
```

Reply:
You want me to create something like 
```python
try:
    return (bear.name.lower() in (
            map(str.lower, list(bear.section['debug_bears']))) or bool(bear.section['debug_bears']))
except ValueError:
    raise ValueError('could not parse setting "debug_bears" into bool or string-list')
```
then John already said not to do so, direclty said to only follow the the things.

Reply:
you want me to do somehting like this ?
```python
    try:
        return (bear.name.lower() in (
                map(str.lower, list(bear.section['debug_bears']))) or
                    bool(bear.section['debug_bears']))
    except ValueError:
        raise ValueError('could not parse setting "debug_bears" into bool or '
                         'string-list')
```

Reply:
I have updated till now what I have got :+1:

Reply:
I made a silly mistake and I will take care of that next time for further update of this reply check [here](https://github.com/coala/coala/pull/5677#discussion_r206724245)